app/domain/queries/find_content.rb

```diff
@@ -61,4 +69,20 @@ def slice_editions
     editions = editions.where('document_type = ?', document_type) if document_type
     editions
   end
+
+  def model_name
+    return Aggregations::SearchLastThirtyDays if date_range == 'last-30-days'
+    return Aggregations::SearchLastMonth if date_range == 'last-month'
+    return Aggregations::SearchLastThreeMonths if date_range == 'last-3-months'
+    return Aggregations::SearchLastSixMonths if date_range == 'last-6-months'
+    Aggregations::SearchLastTwelveMonths
+  end
+
+  def table_name
+    return 'last_thirty_days' if date_range == 'last-30-days'
+    return 'last_months' if date_range == 'last-month'
+    return 'last_three_months' if date_range == 'last-3-months'
+    return 'last_six_months' if date_range == 'last-6-months'
+    'last_twelve_months' if date_range == 'last-year'
```

Comment:
If the `date_range` is none of the expected, I would raise an exception because we are not using the query planner properly.

Reply:
I have defaulted the date_range to `last-30-days` because I think we need a default so that we don't always have to pass a date range, for instance when the user firsts visits the page, but I also raise errors if we don't give a valid date_range